### PR TITLE
Language:  Update simplified Chinese translations & remove unnecessary space

### PR DESCRIPTION
**Pull request by sbzlzh**
@TheOnly8Z would you mind taking a look?

### CODE REVIEW

**Comment by Racpy** [comment]
I think the original zh-tw version is more friendly to Taiwanese people, with correct text and clearer sentences.
Even though some new features are currently untranslated, I will help translate them if I have time.

**Comment by sbzlzh** [comment]
> I think the original zh-tw version is more friendly to Taiwanese people, with correct text and clearer sentences. Even though some new features are currently untranslated, I will help translate and correct text if I have time.

You have a point there. I was using the translations in zh-hans converted to zh-tw by a tool that may not be accurate. Thank you for your time.

**Review by TimGoll — APPROVED** [review]
from my point of view these changes are fine, if you both like the translation, we can merge it

**Comment by sbzlzh** [comment]
I found some of the string translations before zh-tw to be a bit offensive.

**Comment by sbzlzh** [comment]
@TheOnly8Z 觉得1894和1895翻译成什么好

**Comment by sbzlzh** [comment]
I don't have any more problems.

**Comment by TEGTainFan** [comment]
不推荐你使用自动翻译来处理这些文本，如果要用，也请麻烦修复机翻的常见错误，例如一个劲的用敬语_(:з」∠)_

**Comment by sbzlzh** [comment]
all right

**Comment by saibotk** [comment]
@TEGTainFan or @TheOnly8Z if one of you also approves this PR officially we will merge this, but feel free to further suggest changes or discuss. Just letting you know ✌️

**Comment by TimGoll** [comment]
why did you close this?

**Comment by sbzlzh** [comment]
> 你为什么关闭这个？

I got it on GitHub Desktop because I chose the wrong one :)

**Comment by sbzlzh** [comment]
😄

**Comment by sbzlzh** [comment]
@TheOnly8Z There's nothing else wrong with it!

**Comment by TheOnly8Z** [comment]
Most of the localization looks good, I'll make some small amendments later today.

**Comment by sbzlzh** [comment]
> Most of the localization looks good, I'll make some small amendments later today.

Come on, look at you. I'm a little sleepy. I'm a little sleepy.)

**Comment by sbzlzh** [comment]
> > Most of the localization looks good, I'll make some small amendments later today.
> 
> Come on, look at you. I'm a little sleepy. I'm a little sleepy.)

File: lua/terrortown/lang/zh_hans.lua [hunk 1]
```diff
@@ -959,10 +959,10 @@ L.corpse_too_far_away = "这个尸体太远了。"
 L.radio_pickup_wrong_team = "你不能捡起其他队伍的收音机"
 L.radio_short_desc = "武器声音，悦耳动听"
 
-L.hstation_subtitle = "按 [{usekey}] 恢复生命"
+L.hstation_subtitle = "按 [{usekey}] 恢复生命值"
 L.hstation_charge = "剩余充能：{charge}"
 L.hstation_empty = "这个医疗站没有剩余充能"
-L.hstation_maxhealth = "你的生命恢复已满"
+L.hstation_maxhealth = "你的生命值恢复已满"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:965]:
```suggestion
L.hstation_maxhealth = "你的生命值已满"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 2]
```diff
@@ -1488,7 +1488,7 @@ L.help_spawn_editor_spawn_amount = [[
 
 {playerrandom} x 玩家随机位置生成]]
 
-L.equipmenteditor_name_auto_spawnable = "设备在地图中随机产生"
+L.equipmenteditor_name_auto_spawnable = "装备在地图中随机产生"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1491]:
```suggestion
L.equipmenteditor_name_auto_spawnable = "装备可在地图中随机生成"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 3]
```diff
@@ -1617,7 +1617,7 @@ L.help_detective_hats = [[
 
 部分玩家模型默认不支持帽子。你可以在'管理'->'玩家模型'中改变这一点。]]
 
-L.label_roles_credits_award_kill = "Credit reward amount for the kill"
+L.label_roles_credits_award_kill = "击杀后的积分奖励数量"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1620]:
```suggestion
L.label_roles_credits_award_kill = "击杀获得的积分奖励"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 4]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
 L.slot_weapon_class = "职业槽"
 
 -- 2023-10-04
---L.label_voice_duck_spectator = "Duck spectator voices"
---L.label_voice_duck_spectator_amount = "Spectator voice duck amount"
---L.label_voice_scaling = "Voice Volume Scaling Mode"
---L.label_voice_scaling_mode_linear = "Linear"
---L.label_voice_scaling_mode_power4 = "Power 4"
---L.label_voice_scaling_mode_log = "Logarithmic"
+L.label_voice_duck_spectator = "降低观察者语音"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1890]:
```suggestion
L.label_voice_duck_spectator = "观察者语音淡化"
```

Closest thing to a localization to the technical term "audio ducking" seems to be "音频闪避", which literally reads "audio evasion". This seems very obscure and unintuitive, so I elected to use "淡化" (fade out) instead.

File: lua/terrortown/lang/zh_hans.lua [hunk 5]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.label_voice_duck_spectator_amount = "降低观察者语音音量"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1891]:
```suggestion
L.label_voice_duck_spectator_amount = "观察者语音淡化程度"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 6]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.label_voice_scaling = "语音音量缩放模式"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1892]:
```suggestion
L.label_voice_scaling = "语音淡化比例"
```
Literally "voice ducking scale".

File: lua/terrortown/lang/zh_hans.lua [hunk 7]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
---L.label_keyhelper_voice_global = "global voice chat"
---L.label_keyhelper_voice_team = "team voice chat"
---L.label_keyhelper_chat_global = "global chat"
---L.label_keyhelper_chat_team = "team chat"
---L.label_keyhelper_show_all = "show all"
---L.label_keyhelper_disguiser = "toggle disguiser"
---L.label_keyhelper_save_exit = "save and exit"
---L.label_keyhelper_spec_third_person = "toggle third person view"
+L.help_keyhelp = [[
+按键绑定帮助器是用户界面元素的一部分，它总是向玩家显示相关的按键绑定，这对新玩家特别有帮助。按键绑定有三种不同类型：
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1899]:
```suggestion
键位助手是一个向玩家显示相关键位的界面元素，对新玩家特别有帮助。键位分为三种类型：
```
Recommend localizing both keybinding and binding to "键位" as "绑定" is somewhat ambiguous and "键位绑定" is too long.

File: lua/terrortown/lang/zh_hans.lua [hunk 8]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+
+核心：它们包含 TTT2 中最重要的绑定。没有它们，游戏就很难发挥其全部潜力。
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1901]:
```suggestion
核心：包含 TTT2 中最重要的键位。不使用它们的话很难体验游戏的全部潜力。
```

File: lua/terrortown/lang/zh_hans.lua [hunk 9]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+额外：与核心绑定类似，但并非总是需要。它们包含聊天、语音或手电筒等内容。启用这些功能可能会对新玩家有所帮助。
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1902]:
```suggestion
附加：类似核心键位，但并非必要使用。这包含聊天、语音或手电筒等功能。显示这些键位也许会对新玩家有所帮助。
```

File: lua/terrortown/lang/zh_hans.lua [hunk 10]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+装备：有些装备项目有自己的绑定，这些都显示在此类别中。
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1903]:
```suggestion
装备：部分装备有自己的键位，这些都显示在此类别中。
```

File: lua/terrortown/lang/zh_hans.lua [hunk 11]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+
+当记分板可见时，已禁用的类别仍会显示]]
+
+L.label_keyhelp_show_core = "启用始终显示核心绑定"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1907]:
```suggestion
L.label_keyhelp_show_core = "启用始终显示核心键位"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 12]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.label_keyhelp_show_extra = "启用始终显示额外绑定"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1908]:
```suggestion
L.label_keyhelp_show_extra = "启用始终显示额外键位"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 13]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.label_keyhelp_show_equipment = "启用始终显示装备绑定"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1909]:
```suggestion
L.label_keyhelp_show_equipment = "启用始终显示装备键位"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 14]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+
+L.header_interface_keys = "按键绑定帮助器设置"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1911]:
```suggestion
L.header_interface_keys = "键位助手设置"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 15]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.header_interface_wepswitch = "武器切换UI设置"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1912]:
```suggestion
L.header_interface_wepswitch = "武器选择界面设置"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 16]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
-
---L.status_on = "on"
---L.status_off = "off"
-
---L.crowbar_help_primary = "Attack"
---L.crowbar_help_secondary = "Push players"
+L.item_armor_reinforced = "强化装甲"
+L.item_armor_sidebar = "盔甲保护你免受子弹穿透身体。但不是永远。"
+L.item_disguiser_sidebar = "盔甲可防止子弹穿透身体。但不是永远。"
+L.status_speed_name = "速度倍增器"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1944]:
```suggestion
L.status_speed_name = "移速倍率"
```

"Multiplier" is referring to the mathematical operation and not a device.

File: lua/terrortown/lang/zh_hans.lua [hunk 17]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.status_speed_description_good = "你的速度快于常人。物品、装备或效果都会对此产生影响。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1945]:
```suggestion
L.status_speed_description_good = "你的移动速度高于普通。物品，装备或效果可能会影响此值。"
```

File: lua/terrortown/lang/zh_hans.lua [hunk 18]
```diff
@@ -1889,66 +1887,66 @@ L.slot_weapon_extra = "额外槽"
...
+L.status_speed_description_bad = "您的速度比正常速度慢。物品、装备或效果会对此产生影响。"
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1946]:
```suggestion
L.status_speed_description_bad = "你的移动速度低于普通。物品，装备或效果可能会影响此值。"
```

**Review by TheOnly8Z — CHANGES_REQUESTED** [review]

**Comment by sbzlzh** [comment]
@TheOnly8Z It's done. See if there's anything missing.

**Comment by TimGoll** [comment]
I updated your branch for you so you don't have the issue with the closing pullrequest

File: CHANGELOG.md [hunk 19]
```diff
@@ -22,13 +22,16 @@ All notable changes to TTT2 will be documented here. Inspired by [keep a changel
   - Can be disabled in Appearance->Interface
   - Shows binding name when scoreboard is opened
 - Added option to render rotated text on screen (by @TimGoll)
+- Add the required string translations (by @sbzlzh)
+- Improved translation of some Simplified Chinese strings (by @TheOnly8Z)
```
Comment (saibotk) [inline CHANGELOG.md:26]: Can you move those below to the changed section, where you already added an entry?
Also rephrase the first point to something like: `Added all new translation strings`

**Review by saibotk — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 19 above)
Comment (sbzlzh) [reply CHANGELOG.md:26]: ok

**Comment by sbzlzh** [comment]
Is it like this?

File: CHANGELOG.md (hunk 19 above)
Comment (sbzlzh) [reply CHANGELOG.md]: So I didn't make a mistake in modifying it, did I!

**Comment by sbzlzh** [comment]
：）

**Comment by sbzlzh** [comment]
@TheOnly8Z Let's see if there are any other questions.

File: lua/terrortown/lang/zh_hans.lua [hunk 20]
```diff
@@ -1,4 +1,4 @@
--- Simplified Chinese language strings (by 8z & TEGTianFan & 波と一緒に漂う)
+-- Simplified Chinese language strings (by 8z & TEGTianFan)
```
Comment (TheOnly8Z) [inline lua/terrortown/lang/zh_hans.lua:1]:
```suggestion
-- Simplified Chinese language strings (by 8Z & TEGTianFan)
```

**Review by TheOnly8Z — COMMENTED** [review]
Looks good to merge, just capitalize my username here and it should be set.

**Comment by sbzlzh** [comment]
OK :)

**Review by saibotk — APPROVED** [review]

**Pull request merged**